### PR TITLE
Fix Android builds (2020-08-04)

### PR DESCRIPTION
**Pull request by ZhilkinSerg**
#### Summary

```SUMMARY: None```

### CODE REVIEW

**Comment by anothersimulacrum** [comment]
Whoops, I should have known android would have some special stuff here when I was touching it.

File: src/sdltiles.cpp [hunk 1]
```diff
@@ -2210,7 +2210,7 @@ void draw_quick_shortcuts()
             if( touch_input_context.get_category() == "INVENTORY" && inv_chars.valid( key ) ) {
                 Character &player_character = get_player_character();
                 // Special case for inventory items - show the inventory item name as help text
-                hint_text = player_character.inv.find_item( player_character.inv.invlet_to_position(
+                hint_text = player_character.inv.find_item( player_character.inv->invlet_to_position(
```
Comment (kevingranade) [inline src/sdltiles.cpp:2213]:
```suggestion
                hint_text = player_character.inv->find_item( player_character.inv->invlet_to_position(
```

**Comment by ZhilkinSerg** [comment]
Travis is passing Android build now, so I will self-merge to include in the next Jenkins build.

**Comment by cainiaowu** [comment]
Jenkins android build keeps failling, checking the log we could see that it was trying to build both v7a and v8 in the same run, however it didnt have enough time to finish due to 1 hour timeout. 
Maybe we should separate it into 2 runs so that it can finish.

**Comment by ZhilkinSerg** [comment]
Android builds are not a priority - it is enough that they succeed once in a while. There is very limited disk space on Jenkins machine, so splitting Android builds to different jobs is not an option.

**Pull request merged**